content/blog/kubecon_eu_2018.md

```diff
@@ -0,0 +1,62 @@
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+
+If you couldn't make it to KubeCon or attend the talks in person, you can view the CNCF posted videos or download the slide presentations below.
+
+> NATS Intro hosted by Colin Sullivan | [Review Slides](https://schd.ws/hosted_files/kccnceu18/e0/KubeCon%20EU%20NATS%20Intro%20v2.pdf) | [Watch Video](https://www.youtube.com/watch?v=Y9bDY_oE80w&t=1625s)
+
+> NATS Deep Dive hosted by Wally Quevedo | [Review Slides](https://schd.ws/hosted_files/kccnceu18/e8/KubeCon%20EU%20-%20NATS%20Deep%20Dive.pdf) | [Watch Video](https://www.youtube.com/watch?v=MAKYz8oPRyw&t=4s)
+
+Now that the 'work' was out of the way we could relax and enjoy the rest of the conference. Thursday evening, KubeCon + CloudNativeCon EU was welcomed at [Tivoli Gardens](http://www.tivoli.dk/) for some food, drinks, and fun! We had a great time chatting with other conference goers, enjoying the nice weather and rides including a few rollercoasters and a spirited session of bumper cars.
+
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/fluentd?ref_src=twsrc%5Etfw">@fluentd</a> having fun with <a href="https://twitter.com/nats_io?ref_src=twsrc%5Etfw">@nats_io</a> team. (Talking about Nats+TLS cc <a href="https://twitter.com/derekcollison?ref_src=twsrc%5Etfw">@derekcollison</a> go for it!) <a href="https://twitter.com/CloudNativeFdn?ref_src=twsrc%5Etfw">@CloudNativeFdn</a> <a href="https://twitter.com/hashtag/KubeCon?src=hash&amp;ref_src=twsrc%5Etfw">#KubeCon</a> <a href="https://t.co/dmcVfQu51W">pic.twitter.com/dmcVfQu51W</a></p>&mdash; Eduardo Silva (@edsiper) <a href="https://twitter.com/edsiper/status/992083494103846912?ref_src=twsrc%5Etfw">May 3, 2018</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+We are thankful to the Cloud Native Computing Foundation and all those who participated in KubeCon. We sincerely appreciate the community's interest in NATS and know our time spent in Copenhagen was successful to further education about NATS. We look forward to seeing you in December at [KubeCon + CloudNativeCon Seattle 2018](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/)!
+
+
+> Additional KubeCon Resources
+
+> [View All Videos](https://www.youtube.com/playlist?list=PLj6h78yzYM2N8GdbjmhVU65KYm_68qBmo)
+
+> [View Flickr Gallery](https://www.flickr.com/photos/143247548@N03/sets/72157696369579365/with/40046937380/)
+
+
```

Comment:
Can you drop these spaces?

Reply:
which empty lines? at the bottom of the doc?

Reply:
Yep.